docs/ocis/getting-started/_index.md

```diff
@@ -60,9 +64,18 @@ The `latest` tag always reflects the current master branch.
 
 ```console
 docker pull owncloud/ocis
-docker run --rm -ti -p 9200:9200 owncloud/ocis
+docker run --rm -ti -p 9200:9200 -e OCIS_INSECURE=true owncloud/ocis
 ```
 
+{{< hint info >}}
+When you're using oCIS with self-signed certificates, you need to set the environment variable `OCIS_INSECURE=true`, in order to make oCIS work.
+{{< /hint >}}
+
+{{< hint warning >}}
+In order to persist your data, you need to mount a docker volume or create a host bind-mount at `/var/lib/ocis`, for example with: `-v /some/host/dir:/var/lib/ocis`
```

Comment:
please add a warning that this doesn't work on Mac because mac doesn't support xattr via bind mounts

Reply:
do you know about a tracking issue at docker? That would be nice to link

Reply:
i think they said "Using docker on mac is not supported for production"

Docker for linux has no issues with that.